.github/workflows/ci-build.yml

```diff
@@ -39,44 +39,131 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
 
+  e2e-tests:
```

Comment:
This is just a suggestion: It is probably nice to wait until the `lint-and-unit-test` succeeds before starting this one. The e2e tests will likely fail if the unit tests fail.

Reply:
This has been change because the e2e tests take 20 minutes to run. A similar thing has been done in datagateway.